src/category_theory/limits/shapes/kernel_pair.lean

```diff
@@ -0,0 +1,174 @@
+/-
+Copyright (c) 2020 Bhavik Mehta. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Bhavik Mehta
+-/
+import category_theory.limits.shapes.equalizers
+import category_theory.limits.shapes.pullbacks
+import category_theory.limits.shapes.regular_mono
+
+/-!
+# Kernel pairs
+
+This file defines what it means for a parallel pair of morphisms `a,b : R ⟶ X` to be the kernel pair
```

Comment:
```suggestion
This file defines what it means for a parallel pair of morphisms `a b : R ⟶ X` to be the kernel pair
```
I found the commas pretty confusing. Let's not use them inside backticks?

Reply:
Fair enough - do you think it would be better to write `(a,b)` for the kernel pair? I think I had that in some earlier versions